### PR TITLE
User able to navigate site

### PR DESCRIPTION
**Pull request by NegativeKarma**
# PT-link: https://www.pivotaltracker.com/story/show/154699963

### CODE REVIEW

**Review by zuzanna81 — DISMISSED** [review]
In user story you use index page and landing page but you refer to the same path (root_path). What we want is to land on the sign_up or sign_in page when we click Login or Sign up on index page. After that we want to be on the family tree page (where we display the tree), but this is not the part of this story :-) So we need to rewrite user story and add Sign up and Log in buttons to navbar when both PRs are merged.

File: features/step_definitions/basic_steps.rb [hunk 1]
```diff
@@ -18,8 +22,8 @@
   expect(page.current_path).to eq page_path_from(page_name)
 end
 
-Then("I should be redirected to the index page") do
```
Comment (lcegit) [inline features/step_definitions/basic_steps.rb]: Why has this step been deleted?

File: features/step_definitions/basic_steps.rb [hunk 2]
```diff
@@ -18,8 +22,8 @@
...
-  expect(page.current_path).to eq root_path
+When("I click {string} button") do |button|
+  click_button button
```
Comment (lcegit) [inline features/step_definitions/basic_steps.rb:26]: Do we need a button here? Wouldn't it be sufficient with a link?

File: features/user_can_log_in_and_log_out.feature [hunk 3]
```diff
@@ -20,22 +20,22 @@ Feature: User has access to login and logout
   Scenario: Visitor fills in all fields [Happy Path]
     When I fill in "Email" with "lisa@gmail.com"
     And I fill in "Password" with "heythere123"
-    And I click "Log in"
-    Then I should be redirected to the index page
+    And I click "Log in" button
```
Comment (lcegit) [inline features/user_can_log_in_and_log_out.feature:23]: is the button needed?

**Review by lcegit — COMMENTED** [review]
Looks good! Just a few comments regarding the step defs.

File: features/step_definitions/basic_steps.rb (hunk 2 above)
Comment (NegativeKarma) [reply features/step_definitions/basic_steps.rb:26]: both for the log in and and sign up it's a button in the log in and sign up pages

File: features/user_can_log_in_and_log_out.feature (hunk 3 above)
Comment (NegativeKarma) [reply features/user_can_log_in_and_log_out.feature:23]: yes, it's a button otherwise it would click on the link in the navbar

**Pull request merged**